src/plugin.js

```diff
@@ -184,31 +185,41 @@ const initPlugin = function(player, options) {
     if (display.getChild('closeButton')) {
       display.removeChild('closeButton');
     }
-    // Make the error display closeable, and we should get a close button
-    display.closeable(true);
+
     content.className = 'vjs-errors-dialog';
     content.id = 'vjs-errors-dialog';
     content.innerHTML =
-      `<div class="vjs-errors-content-container">
-        <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
-          <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
-          ${details}
-        </div>
-        <div class="vjs-errors-ok-button-container">
+     `<div class="vjs-errors-content-container">
+      <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
+        <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
+        ${details}
+      </div>`;
+
+    // Make the error display closeable, and we should get a close button
+    if (!('dismiss' in error) || error.dismiss) {
```

Comment:
Related to this line, maybe this would be clearer (with some lines removed):

```js
closeable = this.closeable(!('dismiss' in error) || error.dismiss);

if (closeable) {
  // add "OK" button
}
```